spacy_curated_transformers/cli/fill_config_transformer.py

```diff
@@ -0,0 +1,265 @@
+import json
+from enum import IntEnum
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+from spacy import util
+from spacy.cli._util import import_code, init_cli, parse_config_overrides
+from spacy.cli.init_config import save_config
+from typer import Argument as Arg
+from typer import Context
+from typer import Option as Opt
+from wasabi import Printer
+
+from .._compat import has_hf_transformers, has_huggingface_hub
+
+
+@init_cli.command(
+    "fill-config-transformer",
+    context_settings={"allow_extra_args": True, "ignore_unknown_options": True},
+)
+def init_fill_config_transformer_cli(
+    # fmt: off
+    ctx: Context,  # This is only used to read additional arguments
+    base_path: Path = Arg(..., help="Path to the base config file to fill", exists=True, allow_dash=True, dir_okay=False),
+    output_path: Path = Arg("-", help="Path to output .cfg file or '-' for stdout (default: stdout)", exists=False, allow_dash=True),
+    code_path: Optional[Path] = Opt(None, "--code-path", "--code", "-c", help="Path to Python file with additional code (registered functions) to be imported"),
+    transformer_name: Optional[str] = Opt(None, "--name", "-n", help="Name of the transformer pipe whose config is to be filled (default: first transformer pipe)."),
+    # fmt: on
+):
+    """
+    Fetches the hyperparameters of a Curated Transformer model
+    from the Hugging Face Hub and fills in the config with them.
+
+    DOCS: https://spacy.io/api/cli#init-fill-config-transformer
+    """
+
+    overrides = parse_config_overrides(ctx.args)
+    import_code(code_path)
+    init_fill_config_transformer(
+        base_path,
+        output_path,
+        config_overrides=overrides,
+        transformer_name=transformer_name,
+    )
+
+
+CURATED_TRANSFORMER_TO_HF_MODEL_TYPE: Dict[str, str] = {
+    "AlbertTransformer": "albert",
+    "BertTransformer": "bert",
+    "CamembertTransformer": "camembert",
+    "RobertaTransformer": "roberta",
+    "XlmrTransformer": "xlm-roberta",
+}
+
+
+class HfParamSource(IntEnum):
```

Comment:
Why not a regular enum?

Also, the last time I tried, `IntEnum` also works without these tupled-type members? The `IntEnum` documentation also uses regular integers?

Reply:
Changed to `Enum`.